packages/cli/src/utils.ts

```diff
@@ -105,3 +106,55 @@ export function getDeployableOptions({
 
   return [{ config, account }, chain] as const;
 }
+
+export function coerceBigInts<T extends ZodType>(
```

Comment:
@Quazia I'm pretty sure zod has the `z.coerce.bigint()` helper for this purpose but i can see this being useful downstream typing if you don't want to do a full parse

Reply:
If that's enough then maybe we just remove this - idk why but for some reason based on our conversation yesterday I thought we needed this.